etc/services/dcalign.yaml

```diff
@@ -51,6 +51,8 @@ services:
     name: RICH
   - class: org.jlab.service.rtpc.RTPCEngine
     name: RTPC
+  - class: org.jlab.service.mltn.MLDCEngine
```

Comment:
dcalign.yaml is really only used for straight tracks/alignment runs. The new service should be added to data-ai.yaml or data-cv.yaml or data-aicv.yaml.